server/compose/rest.yaml

```diff
@@ -1163,6 +1166,7 @@ endpoints:
       post:
       - { name: records, type: "[]string", title: Records to update }
       - { name: values, type: "types.RecordValueSet", title: Fields to update and their values }
+      - { name: includeAllRecords, type: "bool", title: Flag value for selecting all records }
```

Comment:
I can get behind adding this new parameter; I'm just not sure about us changing the meaning of what `records` does when this is set (when set, it becomes the exclude list). What do you think? Should we have an extra parameter just for excluded records?

Reply:
This would apply to all of affected endpoints